opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -154,6 +154,13 @@ impl LogProcessor for SimpleLogProcessor {
 /// them at a pre-configured interval.
 pub struct BatchLogProcessor<R: RuntimeChannel> {
     message_sender: R::Sender<BatchMessage>,
+
+    // Track dropped logs. We'll log this at shutdown and also emit
```

Comment:
nit: remove the comment about emit as metric that was removed.

Reply:
```suggestion
// Track dropped logs - we'll log this at shutdown
```